decoders/connector/atomsenses/as-201/v1.0.0/payload.js

```diff
@@ -0,0 +1,141 @@
+/**
```

Comment:
Hello @mattAtomsenses, thank you for opening a Pull Request to add a public decoder. We appreciate your effort and contribution to the TagoIO IoT community. I have a few points:

1. The new decoders added must be made in TypeScript and also add type annotation. When changing the payload to TypeScript, don't forget to edit the `connector.jsonc` file to `payload.ts`.

2. This decoder must have a unit test, here is an example of a unit test you can use.
```ts
import { describe, test, expect } from "vitest";

import { decoderRun } from "../../../../../src/functions/decoder-run";

const file_path = "decoders/connector/your-path/payload.js" as const;

describe("Shall not be parsed", () => {
  let payload = [{ variable: "shallnotpass", value: "04096113950292" }];
  payload = decoderRun(file_path, { payload });
  test("Output Result", () => {
    expect(Array.isArray(payload)).toBe(true);
  });

  test("Not parsed Result", () => {
    expect(payload).toEqual([{ variable: "shallnotpass", value: "04096113950292" }]);
  });
});
```

If you can add a unit test for the payload parser, that would be great.